### ISSUE TITLE
[a11y][ShellBar]: Logo is not focusable

### ISSUE THREAD

**Issue by ArtemJJS** [SAP/ui5-webcomponents#11321]
### Bug Description

We have clickable both, SAP logo and primary title, this is a requirement for our application, as we have to provide easy way for the users to reset current state and go to initial user's notifications page.
We found that in case ShellBar "menuItems" are empty - there is no separate focus for SAP Logo and PrimaryTitle and both elements are interpreted as a single element.

We expect the possibility to navigate and click separately on Logo and PrimaryTitle.

### Affected Component

ShellBar

### Expected Behaviour

When Logo and PrimaryTitle are provided and menuItems list is empty - logo and primary title are navigable with keyboard separately

### Steps to Reproduce

1. Shellbar with Logo and PrimaryTitle, but without menuItems
2. Try to navigate to the clickable logo and clickable primary title separately

### Log Output, Stack Trace or Screenshots

Current behavior:

https://github.com/user-attachments/assets/1a993363-1e23-4bb7-bc18-2c4c6f229f7a

Expected exaple (we couldn't avoid empty focus slot after "Notification Center" title):

https://github.com/user-attachments/assets/107da786-368a-4a5a-ae21-0c65ae23b82d

### Priority

None

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Operating System

MacOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

Please have a look at this request for the logo in the ShellBar.

Kind regards, 

Georgi

SAP Development Support

**Comment by dobrinyonkov** [comment]
Hello @ArtemJJS!

This behavior is part of the new design introduced under SAP's UXC initiatives. The logo and title are now combined into a single clickable area referred to as "branding."

Would it be possible to have this entire area navigate users back to the homepage when clicked? 

Otherwise, could you clarify the intended behavior—specifically, what should happen when the primary title is clicked?
Your input would help the design team to evaluate any design changes.

Thanks!
